components/NewsCard.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <v-card class="mx-auto" width="500">
+    <v-img
+      class="align-end"
+      height="200px"
+      :src="news?.urlToImage"
+      @error="imageErrorHandle"
+    >
+      <v-card-title class="bg-gradient">
+        <NuxtLink
+          :to="`/${news?.title}`"
+          class="text-decoration-none text-custom-hover"
+        >
+          #{{ index || '' }} {{ news?.title }}
+        </NuxtLink>
```

Comment:
please explain how this redirection will work and since we don't have any route defined in our system for this, how the page will get displayed?

Reply:
ooh sorry, resolved